common/src/main/java/org/redlance/dima_dencep/mods/rrls/mixins/LoadingOverlayMixin.java

```diff
@@ -75,6 +82,37 @@ private static int replaceAlpha(int color, int alpha) {
                     minecraft.font, ConfigExpectPlatform.reloadText(), i / 2, 70,
                     ConfigExpectPlatform.rgbProgress() ? ThreadLocalRandom.current().nextInt(0, 0xFFFFFF) : -1
             );
+
+            case Type.TEXT_WITH_BACKGROUND -> {
+                if (textWidget == null) {
+                    textWidget = new FocusableTextWidget(i, Component.literal(ConfigExpectPlatform.reloadText()), minecraft.font, 12);
```

Comment:
Can we reset this after the (re)load completes?

Reply:
Okay